test-support/helpers/ember-tooltips.js

```diff
@@ -112,6 +112,11 @@ function getTooltipAndTargetPosition(options = {}) {
   };
 }
 
+/* TODO(Duncan):
+
+Update triggerTooltipTargetEvent() to use getTooltipTargetFromBody
+and change type to side and move side into the options hash */
```

Comment:
why would you change `type to side`? Also, I understand why we'd wnat to move `type` into the hash but that may require a deprecation period. Let's chat about this on slack.